packages/taiko-client/driver/chain_syncer/beaconsync/syncer.go

```diff
@@ -73,9 +73,7 @@ func (s *Syncer) TriggerBeaconSync(blockID uint64) error {
 	}
 
 	fcRes, err := s.rpc.L2Engine.ForkchoiceUpdate(s.ctx, &engine.ForkchoiceStateV1{
-		HeadBlockHash:      headPayload.BlockHash,
-		SafeBlockHash:      headPayload.BlockHash,
-		FinalizedBlockHash: headPayload.BlockHash,
+		HeadBlockHash: headPayload.BlockHash,
```

Comment:
Made this change since i met this error: 

<img width="1222" alt="Pasted Graphic" src="https://github.com/user-attachments/assets/da209068-fd9d-4a32-a24a-a1e29689b0b3">

Is this change safe? cc @mask-pp

Reply:
I'm checking, it's caused by `taiko-geth`.